turbo.json

```diff
@@ -5,7 +5,8 @@
       "dependsOn": ["^build"]
     },
     "build": {
-      "dependsOn": ["^build"]
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**", ".next/**", "!.next/cache/**"]
```

Comment:
heh I knew we weren't actually using Turbo very well – neat if this is the thing that makes it work.